docs/internal/release.rst

```diff
@@ -187,6 +187,15 @@ The workflows are:
 
 These will be triggered by the release script, then must be :ref:`uploaded after release <release-upload-artifacts>`.
 
+Preparing for a Micro Release
+=============================
+
+As stated earlier, a micro release only contains fixes.
+If a bug is found in a release version, the fix should first be applied to the development branch for the corresponding major version and the master branch if they are not the same.
+To start a micro release, cherry pick one or more fixes from the development branch to the release branch.
+Release branches have the form ``branch-DDS-X.Y``.
+The cherry-picked commits should include the relevant news files.
```

Comment:
This should link to (or be merged with) the other section describing micro releases: https://github.com/OpenDDS/OpenDDS/blob/abe12bcb6807b560b58a241f8a88e0d818298cb9/docs/internal/release.rst?plain=1#L340
Also there's a sub command (`--cherry-pick-prs`) to help with cherry picking.